### DISCUSSION TITLE
Option to download subtitles and closed captions?

### DISCUSSION THREAD

**Discussion by Ramingon** [aclap-dev/video-downloadhelper#1408] [Help and Q&A] [answered]
### Your question

No issues with downloading videos, however closed captions and subtitles can't be selected for download, even when they exist for the source. Is this a feature that's going to be implemented please?

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
{
    "used_history_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 12,
    "successfull_dl": 1
}

**Comment by jcfrog** [comment, marked as answer]
Hi,
Nothing is planned but yes, this is a known issue.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

In my experience, closed captions are not separate from the video.  They are burned into the video track of the video.  If closed captions are part of the content, then all you need to do is turn on captions during playback in your favorite media player, which should be VLC.  VDH does not handle closed captions in any way.  Since they are, as I say, part of the video track, VDH gets them as part of the download.  Neither VDH nor you need to do anything to get those.  They just happen.

For subtitles in a separate side file (vtt, srt, etc.), you can get those separately by analyzing the Network Monitor in the Developer Tools.  Usually.  You're on your own doing that.  Like @jcfrog Jérôme (who does work for VDH) said, it's not a feature of VDH at the moment.
